resource_booking/tests/test_backend.py

```diff
@@ -675,3 +675,73 @@ def test_event_show_as_free(self):
         rb_f.partner_id = self.partner.copy()
         with self.assertRaises(AssertionError):
             rb_f.save()
+
+    def test_assign_multiple_humans(self):
+        """
+        It should be possible to assign multiple humans to a booking.
+        """
+        user_3 = self.env["res.users"].create(
+            {
+                "email": "user_3@example.com",
+                "login": "user_3",
+                "name": "User 3",
+            }
+        )
+        r_user_3 = self.env["resource.resource"].create(
+            {
+                "calendar_id": self.r_calendars[0].id,
+                "name": user_3.name,
+                "resource_type": "user",
+                "tz": "UTC",
+                "user_id": user_3.id,
+            }
+        )
+        rbc = self.env["resource.booking.combination"].create(
+            {"resource_ids": [(6, 0, [self.r_users[0].id, r_user_3.id])]}
+        )
+        self.env["resource.booking"].create(
+            {
+                "partner_id": self.partner.id,
+                "start": "2021-03-01 08:00:00",
+                "type_id": self.rbt.id,
+                "combination_id": rbc.id,
```

Comment:
I'm a little confused by this. There's no `resource.booking.type.combination.rel` linking these two. Ought that not ring some alarm bells somewhere?

Reply:
thanks for catching this! fixed.